<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadImg</class>
 <widget class="QWidget" name="ReadImg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: #ee2a24</string>
         </property>
         <property name="text">
          <string>图片信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resolutionLabel">
         <property name="text">
          <string>分辨率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stateLable">
         <property name="text">
          <string>状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="epsLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>位置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color: #ee2a24</string>
         </property>
         <property name="text">
          <string>Waifu2x参数</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>打开Waifu2x</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>噪点等级：3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>放大倍数：2</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>模型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="modelBox">
           <item>
            <property name="text">
             <string extracomment="效果好，不放大">cunet（不放大）</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="效果好，2倍放大">cunet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="写真，速度快，2倍放大">photo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string extracomment="动漫，速度快，2倍放大">anime_style_art_rgb</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>转码模式：GPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resolutionWaifu">
         <property name="text">
          <string>分辨率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sizeWaifu">
         <property name="text">
          <string>大小：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tickLabel">
         <property name="text">
          <string>耗时：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stateWaifu">
         <property name="text">
          <string>状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>铺满高度</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>铺满宽度</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="copyButton">
         <property name="text">
          <string>打开图片工具</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>隐藏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="returePage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>上一章</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>下一章</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="lastPage">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextPage">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>OpenNextEps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modelBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ReadImg</receiver>
   <slot>SwitchModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastPage</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>LastPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>SwitchPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>SwitchPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyButton</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>CopyPicture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPage</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>NextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>OpenLastEps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>returePage</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>ReturnPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>ReadImg</receiver>
   <slot>OpenWaifu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LastPage()</slot>
  <slot>NextPage()</slot>
  <slot>ReturnPage()</slot>
  <slot>SwitchPicture()</slot>
  <slot>CopyPicture()</slot>
  <slot>SwitchWaifu()</slot>
  <slot>OpenWaifu()</slot>
  <slot>ReduceScalePic()</slot>
  <slot>AddScalePic()</slot>
  <slot>OpenLastEps()</slot>
  <slot>OpenNextEps()</slot>
  <slot>SwitchModel()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
